things/api/logging.go

```diff
@@ -264,3 +265,146 @@ func (lm *loggingMiddleware) Identify(ctx context.Context, key string) (id strin
 
 	return lm.svc.Identify(ctx, key)
 }
+
+func (lm *loggingMiddleware) CreateGroup(ctx context.Context, token string, g groups.Group) (gr groups.Group, err error) {
```

Comment:
Please be consistent when using `token` and `key`. I'm OK with using either option. It's a JSON Web **Token** which is Mainflux **Key**, but, using it as `key` here and as `token` there might be interpreted as if it's not the same thing.